ourrevolution/settings.py

```diff
@@ -86,6 +86,16 @@
     'EVENTS_PROMOTE_RECENT_CUTOFF_DAYS',
     14
 ))
+EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ENABLED = bool(int(os.environ.get(
+    'EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ENABLED',
+    0
+)))
+"""Default to -1 so we make sure we don't pull a real cons group if
+config isn't set, since cons groups start at 1"""
+EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID = int(os.environ.get(
+    'EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID',
+    -1
```

Comment:
Can we just default to `None` here instead? That seems better than using -1 which is a bit confusing. Will `None` throw an error when casting to int?

Also why do we need a separate config for enabled or not, maybe we can just use this one config, and assume it's enabled if there's a valid value here, and disabled if there is no valid value here?

Reply:
You can't default to NONE when using an int field, so could remove int check here but that might not be good idea. Thoughts?

I see your point about separate enabled field, perhaps we can assume it's disable if this is missing.

Reply:
Yeah not quite sure best way to do this if `int(None)` throws error. Maybe like this? ```
EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID = if os.environ.get(
    'EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID',
    None
)
if EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID is not None:
    EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID = int(
        EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID
    )
```

Reply:
Another option is just leave it as a string? Are we sure it needs to be an int?